tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -25,6 +27,8 @@ class GraphContext implements Context {
 	 */
 	private FeatureContext $featureContext;
 
+	private $storedUsers = [];
```

Comment:
will this work with the existing user store at core `Provisioning.php`? There we have already a store, cleanup fns. You may be able to reuse those.

Reply:
First I didn't find it in core so I implemented it in ocis. I deleted it and reuse code from core